API_Auto/src/test/java/configReader/UrlReader.java

```diff
@@ -1,22 +1,16 @@
-package APIAssignments;
+package configReader;
 
 import java.io.FileInputStream;
 import java.io.IOException;
 import java.util.Properties;
 
 public class UrlReader {
-
+    //Read Properties file
     public static Properties getProperties() throws IOException {
```

Comment:
Use try catch block to handle the exceptions

Reply:
Done